recipe/meta.yaml

```diff
@@ -1,16 +1,17 @@
 {% set name = "pipenv" %}
-{% set version = "2022.6.7" %}
+{% set version = "2022.7.4" %}
+{% set sha256 = "18420fbae2851d2b9d70d1e00f77a8c55d09704a177fe872a0a2da56e98eb018" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 6ac0e7a8bbe02859d46c53c03d3f33136bbf49c8c7d9a6d818409e17a604eda0
+  sha256: {{ sha256 }}
 
 build:
-  noarch: python
+  # noarch: python
```

Comment:
Could you please add `skip: True  # [py<37]`? See https://github.com/pypa/pipenv/blob/14bc6de4c0fde7fcfc4b3945bdb1d9b61e500944/setup.py#L72

Reply:
done